.vsts-ci.yml

```diff
@@ -0,0 +1,61 @@
+queue:
+  name: Hosted VS2017
+  demands: 
+    - msbuild
+    - visualstudio
+    - vstest
+
+variables:
+  Solution: "Community.Activities.sln"
+  BuildConfiguration: "Release"
+  BuildPlatform: "Any CPU"
+
+steps:
+
```

Comment:
Please add a Build Versioning task. You can inspire from the current Appveyor "Init script" that updates the script. and then add " versioningScheme: "byBuildNumber" as input in nuget pack task.